src/docs/data/builders/color-picker.ts

```diff
@@ -0,0 +1,82 @@
+import { DESCRIPTIONS, KBD } from '$docs/constants';
+import type { APISchema, KeyboardSchema } from '$docs/types';
+import type { BuilderData } from '.';
+
+const builder: APISchema = {
+	title: 'createColorPicker',
+	description: DESCRIPTIONS.BUILDER('color picker'),
+	props: [
+		{
+			name: 'defaultColor',
+			type: 'string',
+			description: 'The default selected color. Should be a hex value, eg: #424242.',
+		},
```

Comment:
It should allow passing in a color store, and a onColorChange fn. Maybe it should be changed to value?

Also, should we maybe allow hsl and rgb? At least we should have derived states, or helpers

Reply:
@TGlide I was thinking about that as well, but having multiple ways of changing the color would quickly lead to circular dependencies I believe, so I think we should stick with one `$value` store where the hex value can be changed and some derived stores for other color values. We can also export functions so users can easily change between different color formats and mention those in the docs.

Reply:
Hmm, why would it lead to circular deps? I think you'd just need a derived that converts it into a single format. So you'd have the value store, that can be anything, and then a derived that tries and converts it. If it's an invalid format, it returns null, or the previously valid input. Do you see that working out?

Reply:
@TGlide I meant that if the user can change the color through multiple stores (one for hex, one for rgb, etc) then it could lead to circular dependencies I believe, because if the hex value changes you would have to update the rgb values and if the rgb values change you would have to update the hex value. But if we just have one `$value` store to control the color and some readable derived stores than it's not a problem.